main.tf

```diff
@@ -1,8 +1,9 @@
 resource "aws_route53_record" "default" {
-  count   = "${var.enabled == "true" ? 1 : 0}"
-  zone_id = "${var.zone_id}"
-  name    = "${var.name}"
-  type    = "${var.type}"
-  ttl     = "${var.ttl}"
-  records = ["${var.records}"]
+  count   = var.enabled == "true" ? 1 : 0
```

Comment:
Terraform 0.12 now supports `bool` type, so I think we should consider switching to that since the module won't be backwards compatible anyways with 0.11